pkg/agent/batch_remote_write_client.go

```diff
@@ -23,12 +23,38 @@ import (
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
 	profilestorepb "github.com/parca-dev/parca/gen/proto/go/parca/profilestore/v1alpha1"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"google.golang.org/grpc"
 )
 
+type metrics struct {
+	writeRawRetries            prometheus.Counter
+	writeRawWithRetriesLatency prometheus.Histogram
+}
+
+func newMetrics(reg prometheus.Registerer) *metrics {
+	var m metrics
+
+	m.writeRawRetries = promauto.With(reg).NewCounter(
+		prometheus.CounterOpts{
+			Name: "parca_agent_batch_writer_retries_total",
+			Help: "Total number of retries when sending WriteRaw gRPC request.",
+		})
+	m.writeRawWithRetriesLatency = promauto.With(reg).NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "parca_agent_batch_writer_latency_seconds",
+			Help:    "Histogram of overall latency when sending WriteRaw gRPC request with retries",
+			Buckets: []float64{.1, .5, 1, 5, 10, 20, 30},
```

Comment:
Since the backoff MaxElapsedTime is configurable(https://github.com/parca-dev/parca-agent/blob/main/pkg/agent/batch_remote_write_client.go#L83), is this bucket values good enough?